dall-e-shell.el

```diff
@@ -419,6 +421,34 @@ ERROR-CALLBACK otherwise."
                                        "KEY-NOT-FOUND")))))
                 "-d" (shell-maker--json-encode request-data))))
 
+(defun dall-e-shell-generate-image (prompt text)
+  "Generate image from text using DALL-E given prompt."
+  (let* ((question (concat prompt text))
+         (png-output (dall-e-shell-post-prompt question)))
+    png-output))
+
+(defun dall-e-shell-generate-image-from-region-0 (prompt)
+  "Generate image from region using DALL-E given prompt."
```

Comment:
No longer needed after consolidating functions (see last comment), but running `M-x checkdock` can help us avoid warnings in docs.

ps. Byte compiling the file is also handy to find other warnings (may not apply in your case).